ReverseNum.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+
+public class ReverseNum {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter a positive three-digit number: ");
+
+        if (!scanner.hasNextInt()) {
+            System.out.println("Please enter a valid number.");
+            return;
```

Comment:
We haven't learned what is `return` and where it can be used. Besides, return is the normal end of the code execution, but we need to terminate the code as I stated in the homework task.